src/test/kotlin/com/pubnub/api/legacy/endpoints/pubsub/PublishTest.kt

```diff
@@ -15,17 +15,23 @@ import com.pubnub.api.CommonUtils.assertPnException
 import com.pubnub.api.CommonUtils.failTest
 import com.pubnub.api.PubNubError
 import com.pubnub.api.PubNubException
+import com.pubnub.api.SpaceId
```

Comment:
What do you think  about adding/modifying integration tests to cover new fields?

Reply:
Added one case